lib/ui/pages/login_page.dart

```diff
@@ -0,0 +1,124 @@
+part of 'pages.dart';
+
+class LoginPage extends StatefulWidget {
+  const LoginPage({Key? key}) : super(key: key);
+
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  TapGestureRecognizer _signupButtonTapRecognizer = TapGestureRecognizer();
+  final _formKey = GlobalKey<FormState>();
+
+  @override
+  void dispose() {
+    _signupButtonTapRecognizer.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: SingleChildScrollView(
+          child: Column(
+            children: [
+              Row(
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: [
+                  IconButton(
+                      onPressed: () {}, icon: Icon(Icons.arrow_back_ios_new)),
```

Comment:
```suggestion
                      onPressed: () {}, icon: const Icon(Icons.arrow_back_ios_new)),
```
one of the examples prefer const with constant constructors